sssom/parsers.py

```diff
@@ -666,6 +689,163 @@ def from_obographs(
+    if filter_by_confident_mappings:
+        df = df[
+            (df['mapAdvice'].str.contains(always_confidence_pattern, regex=True, na=False)) &
+            (~df['mapAdvice'].str.contains(always_confidence_antipattern, regex=True, na=False))]
+
+    # Map mappings
+    mlist: List[Mapping] = []
+    for _, row in df.iterrows():
+        mdict = {
+            'subject_id': f'SNOMED:{row["referencedComponentId"]}',
+            'subject_label': row['referencedComponentName'],
+
+            # 'predicate_id': 'skos:exactMatch',
+            # - mapCategoryId: can use for mapping predicate? Or is correlationId more suitable?
+            #   or is there a SKOS predicate I can map to in case where predicate is unknown? I think most of these
+            #   mappings are attempts at exact matches, but I can't be sure (at least not without using these fields
+            #   to determine: mapGroup, mapPriority, mapRule, mapAdvice).
+            # mapCategoryId,mapCategoryName: Only these in set: 447637006 "MAP SOURCE CONCEPT IS PROPERLY CLASSIFIED",
+            #   447638001 "MAP SOURCE CONCEPT CANNOT BE CLASSIFIED WITH AVAILABLE DATA",
+            #   447639009 "MAP OF SOURCE CONCEPT IS CONTEXT DEPENDENT"
+            # 'predicate_modifier': '???',
+            #   Description: Modifier for negating the prediate. See https://github.com/mapping-commons/sssom/issues/40
+            #   Range: PredicateModifierEnum: (joe: only lists 'Not' as an option)
+            #   Example: Not Negates the predicate, see documentation of predicate_modifier_enum
+            # - predicate_id <- mapAdvice?
+            # - predicate_modifier <- mapAdvice?
+            #   mapAdvice: Pipe-delimited qualifiers. Ex:
+            #   "ALWAYS Q71.30 | CONSIDER LATERALITY SPECIFICATION"
+            #   "IF LISSENCEPHALY TYPE 3 FAMILIAL FETAL AKINESIA SEQUENCE SYNDROME CHOOSE Q04.3 | MAP OF SOURCE CONCEPT
+            #   IS CONTEXT DEPENDENT"
+            #   "MAP SOURCE CONCEPT CANNOT BE CLASSIFIED WITH AVAILABLE DATA"
+            'predicate_id': f'SNOMED:{row["mapCategoryId"]}',
+            'predicate_label': row['mapCategoryName'],
+
+            'object_id': f'ICD10CM:{row["mapTarget"]}',
+            'object_label': row['mapTargetName'],
+
+            # mapping_justification <- mapRule?
+            #   ex: TRUE: when "ALWAYS <code>" is in pipe-delimited list in mapAdvice, this always shows TRUE. Does this
+            #       mean I could use skos:exactMatch in these cases?
+            # mapping_justification <- correlationId?: This may look redundant, but I want to be explicit. In officially downloaded
+            #   SNOMED mappings, all of them had correlationId of 447561005, which also happens to be 'unspecified'.
+            #   If correlationId is indeed more appropriate for predicate_id, then I don't think there is a representative
+            #   field for 'mapping_justification'.
+            # TODO: How to properly get mapping_justification?
```

Comment:
## Issue 1
Not sure how to use `sssom_schema.slots.mapping_justification` here.

## Issue 2
Could take some time to map all the SNOMED mapping fields to SSSOM mapping fields.

---

@matentzn I recommend we merge this now while this is rebased and functional, and we can make improvements to this feature later.